### PR TITLE
chore(webhook): remove docker compose config

### PR DESCRIPTION
**Pull request by ArnaudCalmettes**
Remove `docker_compose` configuration as it is not used anymore.
Update README to reference the current way of testing and deploying the app.

### CODE REVIEW

File: ee/webhooks/README.md [hunk 1]
```diff
@@ -12,26 +12,16 @@ The service has 3 starting modes, split into 3 separate commands:
...
-task install:lint
-task
-```
+## Run linters and tests 
 
-Run the tests for a specific package:
+Run the linters:
 ```
-task tests:local PKG=./pkg/model
+earthly +lint
```
Comment (paul-nicolas) [inline ee/webhooks/README.md:19]: Should we add -P for priviledge ? (for users that do not have them in their configs)

**Review by paul-nicolas — APPROVED** [review]

File: ee/webhooks/README.md (hunk 1 above)
Comment (paul-nicolas) [reply ee/webhooks/README.md:19]: For the +tests, we do not need it for lint sorry

File: ee/webhooks/build.Dockerfile [hunk 2]
```diff
@@ -1,5 +0,0 @@
-FROM ghcr.io/formancehq/base:22.04
```
Comment (gfyrag) [inline ee/webhooks/build.Dockerfile:1]: This file is used by the CI to build releases

Comment (ArnaudCalmettes) [reply ee/webhooks/build.Dockerfile]: I restored it.

File: ee/webhooks/README.md (hunk 1 above)
Comment (ArnaudCalmettes) [reply ee/webhooks/README.md:19]: I added -P for the `earthly -P +tests` command.

**Review by gfyrag — APPROVED** [review]

**Pull request merged**